src/sidebar/oauth-auth.js

```diff
@@ -337,6 +337,27 @@ function auth($http, $rootScope, $window,
     var left   = $window.screen.width / 2 - width / 2;
     var top    = $window.screen.height /2 - height / 2;
 
+    var authWindowSettings = queryString.stringify({
+      left: left,
+      top: top,
+      width: width,
+      height: height,
+    }).replace(/&/g, ',');
```

Comment:
For readability I think it'd be worth moving this into a `strWindowFeatures(obj)` function, or else just adding a code comment, I had to debug this and look into the `window.open` docs to figure out what was going on

Reply:
I added a comment. This legacy API unfortunately has a somewhat suboptimal interface.